lib/import-export-visitor.js

```diff
@@ -794,12 +779,17 @@ function toModuleImport(visitor, source, specifierMap, namespaces) {
   }
 
   const lastIndex = nameCount - 1;
+  let namespaceList = Array.isArray(namespaces) ? namespaces.join(",") : "";
+  const searchExports = ! namespaceList;
```

Comment:
This avoids modifying the provided `namespaces` array and uses the lack of an array as an indicator to search for `exports.`.